js/src/tooltip.js

```diff
@@ -402,27 +413,21 @@ class Tooltip extends BaseComponent {
       return
     }
 
-    if (this._config.html) {
-      if (this._config.sanitize) {
-        content = sanitizeHtml(content, this._config.allowList, this._config.sanitizeFn)
-      }
+    if (this._config.sanitize) {
+      content = sanitizeHtml(content, this._config.allowList, this._config.sanitizeFn)
+    }
```

Comment:
Sanitization will be done for the non HTML content as well?

Reply:
If we find any issue on this, I can revert it, but suppose as it is different option, it is not bad to use it in separate

Reply:
Okay, I just wanted to mention that it does not break anything but changes the behavior. That may or may not be the desired behavior for the users who rely on `this._config.html`. :thinking:

/CC @twbs/js-review